csrc/attention/attention_kernels.cuh

```diff
@@ -85,6 +89,7 @@ inline __device__ float block_sum(float* red_smem, float sum) {
 // Grid: (num_heads, num_seqs, max_num_partitions).
 template <typename scalar_t, typename cache_t, int HEAD_SIZE, int BLOCK_SIZE,
           int NUM_THREADS, vllm::Fp8KVCacheDataType KV_DTYPE,
+          bool IS_LAYER_LEVEL,
```

Comment:
KV_DTYPE is improved to：
  // Layerwise 
  kInt8Group0 = 3,
  // Groupwise 
  kInt8GroupN = 4,

Reply:
It's been quite a long while since I last submitted a PR, and unfortunately, I ended up forgetting that it's possible to complete the fixed changes directly on my local machine before submitting them. Instead, I made the mistake of directly submitting them via the web page, which has led to a significant amount of caches being generated. I sincerely apologize for this oversight. I'll definitely be more careful and follow the correct procedure in the future to avoid such issues.